src/main/java/service/ApiServiceImpl.java

```diff
@@ -0,0 +1,244 @@
+                throw new IOException("httpBody element cannot be null or empty while using method other then GET");
+            }
+
+            connection.setDoOutput(true);
+            DataOutputStream outputStream = new DataOutputStream(connection.getOutputStream());
+            outputStream.writeBytes(httpBody);
+            outputStream.flush();
+            outputStream.close();
+        }
+
+        BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+        String inputLine;
+        StringBuffer response = new StringBuffer();
+        while ((inputLine = in.readLine()) != null) {
+            response.append(inputLine);
+        }
+        in.close();
+
+        return response.toString();
+    }
+
+    public List<Participant> getParticipantById(Long id){
+        ParticipantResponse participantResponse;
+
+        try{
+            participantResponse = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + "participant" + "/" + id,"GET",null),ParticipantResponse.class);
+        }catch (Exception e){
+            throw new CavabungaClientException(e.getMessage());
+        }
+
+        return participantResponse.getData();
+    }
+
+    public List<Participant> getParticipantByUserName(String userName){
+        ParticipantResponse participantResponse;
+
+        try{
+            participantResponse = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + "participant" + "/" + userName,"GET", null),ParticipantResponse.class);
+        }catch (Exception e){
+            throw new CavabungaClientException(e.getMessage());
+        }
+
+        return participantResponse.getData();
+    }
+
+    public List<Participant> getAllParticipants(){
+        ParticipantResponse participantResponse;
+
+        try{
+            participantResponse = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + "participant" + "/","GET",null) , ParticipantResponse.class);
+        }catch (Exception e){
+            throw new CavabungaClientException(e.getMessage());
+        }
+
+        return participantResponse.getData();
+    }
+
+    public void deleteParticipantById(Long id){
+        Response response;
+
+        try {
+            response = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + "participant" + "/" + id, "DELETE", null), Response.class);
+        }catch (Exception e){
+            throw new CavabungaClientException(e.getMessage());
+        }
+    }
+
+    public void saveParticipant(Participant participant){
+        Response response;
+
+        try{
+            if(participant.getId() == null){
+                response = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + participant, "POST", mapper.writeValueAsString(participant)), Response.class);
```

Comment:
https://stackoverflow.com/questions/3730002/code-line-wrapping-how-to-handle-long-lines